processor/configuration/src/main/resources/schemas/sources/aws_sqs_source_0.1.json

```diff
@@ -111,7 +111,10 @@
       "title": "Protocol",
       "description": "The underlying protocol used to communicate with SQS",
       "type": "string",
-      "example": "http or https",
+      "enum": [
+        "http",
+        "https"
+      ],
```

Comment:
@jomarko 
For the record, this file (like all the other RHOC based sinks/sources) is "produced" (manually) from RHOC catalog so we should not change it and ask them to update if necessary. For example this file is derived from [this other](https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/blob/main/etc/kubernetes/manifests/base/connectors/connector-catalog-camel-aws/aws_sqs_sink_0.1.json#L99-L105)

Reply:
@danielezonca ok, so what is your point please? Should I revert the changes? They are quite usefull for RHOSE UI.

Reply:
To be fair @danielezonca if you look at the history of the files they have already had major modification by RHOSE to support features desired in the UI. The JSON schema's may have originated from RHOC but they are very different in RHOSE. See https://github.com/jomarko/sandbox/commit/8c0aef5d5bbbdc07440c382024bfd3bfe3fb98f3 and https://github.com/jomarko/sandbox/commit/8e0b58e985f1c3c14d08ed05460597d29530c328 for examples.

Reply:
@manstis 
Yes let's agree with RHOC team how to proceed for similar changes 👍 
Btw the first PR is not touching the `something_sink_0.1` files so it is safe, only the second one is changing the validation part (with regexpr)